### PR TITLE
Run clang-format over poco

### PR DESCRIPTION
**Pull request by rschu1ze**
```bash
$ utils/check-style/./format.sh base/poco/
$ utils/check-style/./check-style
```

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: base/base/Decimal.h [hunk 1]
```diff
@@ -20,25 +21,25 @@ using Decimal128 = Decimal<Int128>;
...
-    std::is_same_v<T, Decimal32>
-    || std::is_same_v<T, Decimal64>
-    || std::is_same_v<T, Decimal128>
-    || std::is_same_v<T, Decimal256>
-    || std::is_same_v<T, DateTime64>;
+concept is_decimal = std::is_same_v<T, Decimal32> || std::is_same_v<T, Decimal64> || std::is_same_v<T, Decimal128>
+    || std::is_same_v<T, Decimal256> || std::is_same_v<T, DateTime64>;
 
 template <class T>
-concept is_over_big_int =
```
Comment (alexey-milovidov) [inline base/base/Decimal.h:31]: Not in our files, please.

**Review by alexey-milovidov — DISMISSED** [review]
.

File: base/base/Decimal.h (hunk 1 above)
Comment (rschu1ze) [reply base/base/Decimal.h]: Done

**Review by yakov-olkhovskiy — APPROVED** [review]

**Comment by rschu1ze** [comment]
Thanks. Let's merge before some merge conflict destroys this 100.000 LOC piece of art. Test failure (test_cluster_copier) is known and unrelated.

**Pull request merged**